ts/parts/Tooltip.ts

```diff
@@ -23,7 +23,10 @@ declare global {
             tt?: SVGElement;
         }
         interface TooltipFormatterCallbackFunction {
-            (this: TooltipFormatterContextObject): (false|string|Array<string>);
+            (
+                this: TooltipFormatterContextObject,
+                tooltip?: Tooltip
```

Comment:
This new parameter requires a change in the corresponding doclet for `Highcharts.TooltipFormatterCallbackFunction` as well.

Reply:
Thanks! Added, let me know if anything else

Reply:
Now I think.. `tooltip?: tooltip` or perhaps `tooltip: tooltip` ?

Reply:
I am unsure (would tend to optional) and actually I see a broader problem. We do not have Tooltip as an official API class. So you are referencing something, that is not documented.